src/sssom/parsers.py

```diff
@@ -27,8 +27,11 @@
     CONFIDENCE,
     CURIE_MAP,
     DEFAULT_MAPPING_PROPERTIES,
+    JSON_CONTEXT_KEY,
```

Comment:
A key like this which is part of a format (JSON-LD, rather than a specific data model like SSSOM) does IMO not necessarily have to be places in a variable; I don't think this is an issue, just saying that I assume there will be some rule against it.

Reply:
I can revert it back if there's some rule against it. I had no idea.

Reply:
I dont really now; You would have to google or ask chatgpt! I would be interested in that, because it comes up in so many projects. Basically: When is it better to use strings as keys vs static variables.